app/views/papers/_index.html.erb

```diff
@@ -32,8 +32,8 @@
       <small>
         <%= paper.authors.map(&:name).join(', ') %>
         <i>
-          in Journal of Reserved
-          (<%= paper.published_at %>)
+          <%= "in " + paper.publication.name.split.map(&:capitalize).join(' ') unless paper.publication.nil? %>
```

Comment:
Let's do this in CSS instead https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform#Values